### ISSUE TITLE
There is no code of conduct

### ISSUE THREAD

**Issue by y-martinez** [Aman1440/Practice-code-repo#127]
In the readme explanations, you say that follow a code of conduct, but it doesn't have the link.

**Comment by Aman1440** [comment]
yep, there is a broken link which needs to be fixed, though there is a coc in the repo. Thanks for pointing it out.
